requirements-test.txt

```diff
@@ -1,7 +1,5 @@
 flake8==4.0.1
-pep8-naming
 coverage==6.3
-jsonschema==3.2
-safety
 liccheck==0.6.2
 sphinx==7.2.6
+deptry
```

Comment:
@sasha-dresden can you pin a version for `deptry`?

Reply:
Pinned the version. Also I found an [extension](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) that sorts lines. Doesn't do it automatically but really easy to use via vscode commands.